docs/reference/module-api/module-overview.md

```diff
@@ -527,24 +545,9 @@ To maintain the same application, they should return the same type of response a
 
 ### `restApiRoutes(self)`
 
-Add a custom REST API for a module. The `restApiRoutes` function takes the module as an argument and returns an object of functions that map to the standard REST API request types. Route functions may be asynchronous (async).
+Add a custom REST API for a module. The `restApiRoutes` function takes the module as an argument and returns an object of properties that map to the standard REST API request types.
 
-If you simply wish to add to the existing behavior of the REST API routes, see [`extendRestApiRoutes`](#extendrestapiroutes-self).
```

Comment:
I reordered a few of these to be consistent, so it looks like more changed than really was.